src/test/java/com/google/api/codegen/testdata/discoveries/mvvm_nodejs/nodejs_translate.v2.json.baseline

```diff
@@ -14,38 +14,24 @@
 var google = require('googleapis');
 var translate = google.translate('v2');
 
-authorize(function(authClient) {
-  var request = {
-    // The text to detect
-    q: [],  // TODO: Update placeholder value.
-
-    auth: authClient
-  };
-
-  translate.detections.list(request, function(err, response) {
-    if (err) {
-      console.log(err);
-      return;
-    }
-
-    // TODO: Change code below to process the `response` object:
-    console.log(JSON.stringify(response, null, 2));
-  });
-});
+var apiKey = ''; // TODO: Update placeholder with desired API key.
```

Comment:
I like the original. Comparatively, "your" suggests that only one exists, and "the appropriate" suggests that only one is correct, whereas in reality a user may have their own reasons for choosing one or another.